bin/post_compile

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-python manage.py sass static/scss/style.scss static/css/style.css -t compressed
+#python manage.py sass static/scss/style.scss static/css/style.css -t compressed
```

Comment:
So this is probably the place we'll have to call npm install and build I guess? If so we'll need to check what is required to have node/npm available on the scalingo instance. We'll also need to watch how it impacts our already long deploy procedure...

Reply:
Yes still not commited